src/views/Vote/Vote.vue

```diff
@@ -54,7 +63,11 @@ export default class Vote extends Vue {
   private getVoting: any;
 
   public created() {
-    this.initScore = Number(this.$route.query.vote);
+    if (this.$route.query.vote) {
+      this.initScore = Number(this.$route.query.vote);
```

Comment:
If vote comes from the URL why dont you declare it as a property and put `props: true` in the router, you may get rid of this code then.